Compiling refact-problematic with gfortran fails

Hello,

I have recently made a fresh clone of the `refac-problematic` branch and tried to compile it in my Ubuntu 20.04 box. I am using gfortran 9.3.0 with openmpi. Compilation is failing with the following error:
```
/home/ramon/Documents/Work/NETHERLANDS/OPT/champ_ref/src/vmc/error.f:5: Error: Can't open included file 'mpif.h'
make[2]: *** [src/vmc/CMakeFiles/shared_objects.dir/build.make:1646: src/vmc/CMakeFiles/shared_objects.dir/error.f.o] Error 1
```
As a workaround, I forced the Fortran compiler to be mpif90 by setting the Cmake variable:
```
set(CMAKE_Fortran_COMPILER /usr/bin/mpif90)
```
Perhaps the CMakeLists.txt should take this situation into account?

--- a/README.md
+++ b/README.md
@@ -179,7 +179,7 @@ Here are a couple of recipes for commonly used computing facilities, which can b
 		mpirun -s all -np "n process" -machinefile "machinefile"
 		```
 * **Ubuntu desktop**:  
-	- Ubuntu 18.04:
+	- Ubuntu 18.04:  
 		Install the required packages:
 		```
 		sudo apt install gfortran openmpi-bin gawk libblacs-mpi-dev liblapack-dev

